token/cli/src/main.rs

```diff
@@ -385,6 +397,59 @@ async fn command_create_token(
     })
 }
 
+async fn command_set_interest_rate(
+    config: &Config<'_>,
+    token_pubkey: Pubkey,
+    rate_authority: Pubkey,
+    rate_bps: i16,
+    bulk_signers: Vec<Arc<dyn Signer>>,
+) -> CommandResult {
+    println_display(
+        config,
+        format!(
+            "Setting Interest Rate for {} to {} bps",
+            token_pubkey, rate_bps
+        ),
+    );
+
+    let minimum_balance_for_rent_exemption = if !config.sign_only {
+        config
+            .program_client
+            .get_minimum_balance_for_rent_exemption(Account::LEN)
+            .await?
+    } else {
+        0
+    };
```

Comment:
this is unnecessary, no account is being created so the arg passed to `handle_tx` can always be zero. however we should do basic sanity checks inside the `if !config.sign_only` block. off the top of my head i would... fetch mint, check that its owned by token22, check that it has the interest extension, check that the rate authority is correct. you dont need to write this, i can add it after if you prefer